crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -484,6 +491,21 @@ where
             .write(batch)
             .map_err(|e| DatabaseError::Other(e.into()).into())
     }
+
+    // use delete_range to delete all keys in a column
+    fn delete_all(&self, column: Self::Column) -> StorageResult<()> {
```

Comment:
Maybe we want to add a unit test to verify the behaviour of removing leys with different length.